packages/shared/src/components/search/SearchReferralBanner.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement } from 'react';
+import { Button } from '../buttons/Button';
+import { KeyReferralIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { ReferralCampaignKey, useFeatureCampaign } from '../../hooks';
+
+export function SearchReferralBanner(): ReactElement {
+  const { referralCampaign, canInvite } = useFeatureCampaign({
+    campaignKey: ReferralCampaignKey.Search,
+  });
+
+  if (!canInvite) {
+    return null;
+  }
+
+  const { availableCount, noKeysAvailable } = referralCampaign;
+
+  return (
+    <div className="flex flex-col gap-4 p-4 w-full rounded-14 bg-theme-overlay-from">
+      <KeyReferralIcon size={IconSize.Large} className="mr-2" />
+      <h2 className="font-bold typo-title2">
+        {noKeysAvailable
```

Comment:
Where did this copy come from?
Can't see if in design so just missing the context here.
I mainly think it's incomplete as for the button for instance

Reply:
Should be from this message: https://dailydotdev.slack.com/archives/C05NRH3U31C/p1697443536775929?thread_ts=1697387678.267679&cid=C05NRH3U31C